doc/nrf/protocols/matter/getting_started/adding_clusters.rst

```diff
@@ -399,3 +401,270 @@ To check if the sensor device is working, complete the following steps:
+            * ``name`` - The name of the attribute.
+
+         * ``<command>`` - A command definition within the cluster.
+
+            * ``source`` - Specifies whether the command originates from the client or server.
+            * ``code`` - A unique identifier for the command within the cluster.
+            * ``name`` - The name of the command.
+            * ``optional`` - Indicates whether the command is optional.
+            * ``disableDefaultResponse`` - Indicates whether the default response to the command is disabled.
+            * ``response`` - The name of the response command, if any.
+            * ``description`` - A brief description of the command's purpose and functionality.
+            * ``arg`` - An argument for the command, specifying its name and type.
+
+         * ``<event>`` - An event definition within the cluster.
+
+            * ``source`` - Specifies whether the event originates from the client or server.
+            * ``code`` - A unique identifier for the event within the cluster.
+            * ``name`` - The name of the event.
+            * ``optional`` - Indicates whether the event is optional.
+            * ``description`` - A brief description of the event's purpose and functionality.
+            * ``arg`` - An argument for the event, specifying its name and type.
+
+         For example, the following XML code defines a cluster with one attribute, one command, and one event:
+
+         .. code-block:: xml
+
+            <?xml version="1.0"?>
+            <cluster>
+               <domain>General</domain>
+               <name>MyNewCluster</name>
+               <code>0xFFF11234</code>
+               <define>MY_NEW_CLUSTER</define>
+               <description>The MyNewCluster cluster showcases a cluster manufacturer extensions</description>
+               <attribute side="server" code="0x0000" define="FLIP_FLOP" type="boolean" writable="true" default="false" optional="false">MyAttribute</attribute>
+               <command source="client" code="0x02" name="MyCommand" response="MyCommandResponse" optional="false">
+                     <description>
+                        Command that takes two uint8 arguments and returns their sum.
+                     </description>
+                     <arg name="arg1" type="int8u"/>
+                     <arg name="arg2" type="int8u"/>
+               </command>
+               <event source="server" code="0x01" name="MyEvent" optional="false">
+                     <description>
+                        Event that is generated by the server.
+                     </description>
+                     <arg name="arg1" type="int8u"/>
+               </event>
+            </cluster>
+
+   ..
+
+   .. note::
+      The descriptions of the elements above show only the basic functionality.
+      To see the full list of available elements for each part of the XML file, refer to the Matter Specification.
```

Comment:
Link to the Matter spec?

Reply:
It is not so easy. Matter spec can be obtained on a user's request, and it is not visible globally.